dashboard/helpers/helpers.go

```diff
@@ -12,4 +17,19 @@ func CheckErrorWithMessage(err error, message string) {
 	if err != nil {
 		panic(message)
 	}
-}
\ No newline at end of file
+}
+
+// GenerateRandomToken generates a random token of length
+func GenerateProjectToken(length int) (string, error) {
+	if length%2 != 0 {
```

Comment:
interesting. I did not understand. Why modulo 2?

Add some explanation in the form of comments please.